html/changelogs/Incoming5643-slimechanges.yml

```diff
@@ -0,0 +1,6 @@
+author: Incoming5643
+
+delete-after: True
+
+changes: 
+  - rscadd: Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encoraged to investigate."
```

Comment:
I'll be updating the wiki guide too, but yeah, part of getting people to actually use this will be giving them the items (which do have very "this is what they do" descriptions) and saying "you already paid the plasma for this, why not try it?